content/docs/1.1.0/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -11,19 +11,29 @@ This command gives users the ability to restore a backup to a `raw` image or a `
 
 3. Specify the host path of output file by modifying field `hostpath` of volume `disk-directory`. By default the directory is `/tmp/restore/`.
 
-4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://backupbucket@us-east-1/backupstore?backup=backup-bd326da2c4414b02&volume=volumeexamplename`. Do not delete `''`.
+4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://<your-bucket-name>@<your-aws-region>/backupstore?backup=<backup-name>&volume=<volume-name>`.
+
+    - `<backup-name>` and `<volume-name>` can be retrieved from backup.cfg stored in the backup destination folder, e.g. `backup_backup-72bcbdad913546cf.cfg`. The content will be like below: 
+
+        ```json
+        {"Name":"backup-72bcbdad913546cf","VolumeName":"volume_1","SnapshotName":"79758033-a670-4724-906f-41921f53c475"}
+        ```
 
 5. Set argument `output-file` by replacing `<OUTPUT_FILE>`, e.g. `volume.raw` or `volume.qcow2`.
 
 6. Set argument `output-format` by replacing `<OUTPUT_FORMAT>`. The supported options are `raw` or `qcow2`.
 
-7. Set the S3 Credential Secret by replacing `<S3_SECRET_NAME>`, e.g. `minio-secret`. 
+7. Set argument `longhorn-version` by replacing `<LONGHORN_VERSION>`, e.g. `v{{< current-version >}}`
```

Comment:
We don't have this in the doc of existing releases, but instead in the upcoming release only (1.4.0, 1.3.3, and 1.2.7) because all new updates are just for each branch head (master, 1.3.x, 1.2.x).

So, please only keep this line to 1.4.0, 1.3.3, and 1.2.7 doc only.

Reply:
I have removed it and only keep in 1.4.0, 1.3.3, and 1.2.7 document files.